src/chime.clj

```diff
@@ -83,79 +83,3 @@
     (fn cancel! []
       (a/close! ch)
       (reset! !cancelled? true))))
-
```

Comment:
All comments map 1:1 to tests, i.e. no useful testable properties have been lost.